src/css/footer.css

```diff
@@ -0,0 +1,267 @@
+.footer {
+    border-radius: 30px;
+    background: #0c0c0c;
+    max-width: 335px;
+}
+
+.logo-soc-tel-wrapper {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+}
+
+.footer-logo {
+    font-weight: 500;
+    font-size: 16px;
+    letter-spacing: -0.02em;
+    color: #fbfbfb;
+}
+
+.footer-soc-list {
+    display: flex;
+    gap: 10px;
+}
+
+.footer-soc-item {
+    border: 1px solid rgba(251, 251, 251, 0.1);
+    border-radius: 10px;
+    padding: 8px;
+    width: 34px;
+    height: 34px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.footer-soc-link {}
+
+.footer-soc-icon {
+    fill: #fbfbfb;
+
+    transition: fill 250ms ease;
+}
+
+.footer-soc-icon:hover,
+.footer-soc-icon:focus {
```

Comment:
in the UI KIT when hovering, the fill is different, not blue

 (not white???  #fffff)